README.md

```diff
@@ -92,6 +92,14 @@ yarn --version // Should be at least 1.4.2
 ```bash
 git clone https://github.com/paritytech/fether
 cd ./fether
+```
+
+### Update Yarn and install dependencies
+
+```bash
+npm install --global yarn;
+brew upgrade yarn;
```

Comment:
There's a mention to require yarn 1.4.2 above, and that is enough imo. A link to yarn website could be good too.

But the actually command to install yarn, I'd like to avoid. Moreover, you're installing yarn twice here, and both installs could actually not give the same version of yarn.